samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1058,6 +921,334 @@ or
+Unless you are using `Just-in-Time provisioning <nRF Cloud Just-In-Time-Provisioning_>`_, you need to sign these credentials yourself using a CA certificate you create.
+This is referred to as your self-signed CA certificate.
+
+To create your self-signed CA certificate:
+
+1. :ref:`Download and install <nrf_cloud_multi_service_install_nrf_utils>` the `nRF Cloud Utilities <nRF Cloud Utilities documentation_>`_ repository.
+#. Use the nRF Cloud Utilities :file:`create_ca_cert.py` Python script to generate the certificate:
+
+   .. code-block:: console
+
+      python3 create_ca_cert.py -c US -f self_
+
+   Remember to set ``-c`` to your two-letter country code.
+   See the `Create CA Cert <nRF Cloud Utilities Create CA Cert_>`_ section in the nRF Cloud Utilities documentation for more details.
+
+   You should now have the following three files:
+
+     * :file:`self_<self_cert_serial>_ca.pem`
+     * :file:`self_<self_cert_serial>_prv.pem`
+     * :file:`self_<self_cert_serial>_pub.pem`
+
+   Where ``<self_cert_serial>`` is your self-signed CA certificate's serial number in hex.
+   These three files are your self-signed CA certificate and private/public keypair.
+   You will use them to sign device credentials.
+
+   If you were directed here as part of other instructions, proceed to the next step of those instructions.
+
+   .. note::
+      You only need to generate these three files once.
+      You can be use them to sign as many device credentials as you need.
+
+.. _nrf_cloud_multi_service_create_device_cred_locally:
+
+Generating device credentials locally
+=====================================
+
+To generate and sign a device credential locally using your :ref:`self-signed CA certificate <nrf_cloud_multi_service_create_self_signed_ca>`, perform the following steps:
+
+1. Create a globally unique `device ID <nRF Cloud Device ID_>`_ for the device.
+
+   The ID can be anything you want, as long as it is not prefixed with ``nrf-`` and is globally unique.
+   Alternatively, if your device is an LTE development kit from Nordic Semiconductor, you can use ``nrf-<IMEI>`` where ``<IMEI>`` is the IMEI of the device.
+   See `nRF Cloud Device ID`_ for details.
+
+   Each device should have its own device ID, and you must use it exactly for all other actions involving that device, otherwise onboarding will fail.
+   This ID is referred to in other steps using the term *device_uuid*.
+
+#. Navigate to the folder where you :ref:`created your self-signed CA certificate <nrf_cloud_multi_service_create_self_signed_ca>`, and use the :file:`create_device_credentials.py` Python script :ref:`you installed <nrf_cloud_multi_service_install_nrf_utils>` to generate a self-signed certificate for the device:
+
+   .. parsed-literal::
+      :class: highlight
+
+      python3 create_device_credentials.py -ca "self\_\ *self_cert_serial*\ \_ca.pem" -ca_key "self\_\ *self_cert_serial*\ \_prv.pem" -c US -cn "\ *device_uuid*\ " -f cred\_
```

Comment:
Calling it _device_uuid_ is not the best way to refer to the device's ID.  In other documents call it the device id.

In other words, you are unnecessarily overloading the term UUID in this case.